tests/unit/Domain/Entity/Entity/MetaDataTest.php

```diff
@@ -46,7 +46,7 @@ public function test_it_throws_exception_on_invalid_acs_location()
     public function test_it_throws_exception_on_double_acs_locations()
     {
         $data['data'] = json_decode(file_get_contents(__DIR__.'/fixture/read_response_double_acs_location.json'), true);
-        $this->expectExceptionMessage('Double acs locations. Expected unique locations');
+        $this->expectExceptionMessage('Double acs locations. Expected unique locations for entity: ' . $entityId);
```

Comment:
`$entityID` is not defined in this test. The test aims to verify that the correct exception message is generated. So you need to put the actual entityID from the testdata in the line above into the message.

Reply:
I already thought so. I seemed counterintuitive to use a hardcoded value.

Reply:
For a test it's not so problematic since both the testdata and the test itself are closely coupled.